plugins/Marketplace/lang/en.json

```diff
@@ -84,6 +84,7 @@
         "PriceExclTax": "%1$s %2$s excl. tax.",
         "PriceFromPerPeriod": "From %1$s \/ %2$s",
         "Reviews": "Reviews",
+        "RichMenuIntro": "Explore and purchase essential plugins for your dashboard.",
```

Comment:
Did you check if the translations of the removed categories and page are still in use somewhere else? Otherwise they could be removed in that file...

Reply:
I've reinstated the paid plugins widget as it's used in other places (or at least the UI tests indicate that), and just removed the reporting menu category itself, so I think the translations can stay. Haven't done a thorough sweep though.